util/chplspell

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env bash
+
+# Copyright 2004-2016 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Wrapper script around scspell for three reasons:
+# 1. Pass the .scspell/dictionary, and the --relative-to $CHPL_HOME; and
```

Comment:
This line does not parse well for me at all.  I think what it is trying to say is that it is using these as arguments to scspell.  Would you mind being a little bit more explicit instead of concise?